data/json/items/tool/deployable.json

```diff
@@ -3,13 +3,14 @@
     "id": "brazier",
     "type": "TOOL",
     "name": "brazier",
-    "name_plural": "brazier",
-    "description": "A metal stand used to contain a fire.",
+    "description": "A large metal stand used to contain a fire.  Fires set in a brazier will not spread to surrounding flammable objects.",
```

Comment:
Technically, only a **properly installed** brazier prevents fire spreading but let's assume that user knows a brazier works (and will install it, not just drop).